.github/CONTRIBUTING.md

```diff
@@ -8,7 +8,7 @@ Project communication occurs primarily and intentionally via our project [Slack]
 
 However, before you ask in Slack "what can I contribute to", be sure to keep reading this document for the answer to your question. :-)
 
-Also, please avoid use of the `@here` command in Slack, as you will be sending a notification to nearly 600 people. Just post your question and someone will respond soon.
+Also, please avoid use of the `@everyone` and `@channel` commands in Slack, as you will be sending a notification to nearly 600 people. Just post your question and someone will respond soon.
```

Comment:
This looks good, but maybe also include a note about `@here` something like:

> While `@here` will notify only people who are in the room and active, it's best to use this only if you have an announcement that everyone in the channel really needs to here.

Reply:
Sure!